civitai-downloader-v2/src/ui/progress.py

```
@@ -0,0 +1,563 @@
+        
+        if task.metrics.rate > 0:
+            print(f"   Rate: {task.metrics.rate:.2f} items/second")
+        
+        if task.metadata:
+            print(f"   Details: {task.metadata}")
+    
+    def print_overall_summary(self) -> None:
+        """Print overall progress summary."""
+        overall = self.tracker.get_overall_progress()
+        
+        print(f"\n🌟 Overall Progress Summary")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)